control/cli.py

```diff
@@ -229,15 +227,6 @@ def get_actions(act_list):
             acts += ", '" + a["name"] + "'"
         return acts[2:]
 
-    def format_adrfam(self, adrfam):
```

Comment:
@baum why did you remove this formatting? This has nothing to do with protobuf cleanup. I wanted the output to have IPv4 and IPv6 which is the normal way to display address families. I don't understand why you insist on changing that.